### PR TITLE
Add coding standards

### PR DESCRIPTION
**Pull request by SteveRosam**
## Description

Adds coding standards to the contribution guide.

### CODE REVIEW

**Review by peter-quix — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -39,3 +39,10 @@ In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susa
 6. Push changes to your fork.
 7. Open a PR in our repository and follow the PR template so that we can efficiently review the changes.
 
+## Coding standards
+
+We like to keep the code as consistent as possible, so please familiarize yourself with the existing style of the project, module of function you are updating and with the [PEP 8 - Style Guide for Python Code](https://peps.python.org/pep-0008/) if you're contributing to the Python code base or the [Microsoft&reg; C# Coding Conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions) when contributing to the C# code base.
```
Comment (tbedford) [inline CONTRIBUTING.md]: > We like to keep the code as consistent as possible, so please familiarize yourself with the existing style of the project, module of function you are updating and with the [PEP 8 - Style Guide for Python Code](https://peps.python.org/pep-0008/) if you're contributing to the Python code base or the [Microsoft&reg; C# Coding Conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions) when contributing to the C# code base.

I would reword as...

To keep the code as consistent as possible, please familiarize yourself with the existing style of the project. If you're contributing to the Python code base, follow the [PEP 8 - Style Guide for Python Code](https://peps.python.org/pep-0008/).  If contributing to the C# code base, follow the [Microsoft&reg; C# Coding Conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions).

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -39,3 +39,10 @@ In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susa
...
+
+>  Consistency within a project is more important. Consistency within one module or function is the most important.
+
+Note that the biggest deviation from the Microsoft&reg; standards is that we don't use `_` for private members, rather we simply use `this.abc`.
```
Comment (tbedford) [inline CONTRIBUTING.md]: > Note that the biggest deviation from the Microsoft&reg; standards is that we don't use `_` for private members, rather we simply use `this.abc`.

Be more directive...

Note that the biggest deviation from the Microsoft&reg; standards is that this project does not use `_` to indicate private members. Instead, you should use the format `this.abc`.

**Review by tbedford — CHANGES_REQUESTED** [review]
Some minor suggestions on wording.

**Comment by SteveRosam** [comment]
Updated @tbedford

**Review by tbedford — APPROVED** [review]
